mentorship/gsod/2022/project-ideas.md

```diff
@@ -5,16 +5,84 @@ which Keptn contributors have proposed for this year.
+Create excellent instructions for installing Keptn CLI and Keptn on this new cluster.
+Confirm that one set of instructions works for all k8s implementations and all “Keptn curious” projects we provide. 
+If necessary, create multiple instruction sets, clearly defining when each is appropriate.
+
+Also, we propose to create and test instructions for installing a Keptn “playground” in an existing k8s cluster for study and demos. 
+The goal here is to provide an environment where someone can try Keptn out with their existing data without risking damage to an existing production system (such as Prometheus, Dynatrace, DataDog, etc.)
+
+For this deliverable, instructions should be pedagogical rather than pure cookbook:
+provide enough explanation that the user gains some understanding of what they are doing.
+Include information for verifying each step and some troubleshooting information or links.
+
+#### New example projects
+
+Create 5-6 projects that introduce key Keptn capabilities for the Keptn curious to explore. 
+We have a number of existing tutorials to use as starters or inspiration. 
+Note that many of the existing tutorials use the [sockshop project](https://github.com/keptn-demo/sockshop) as a base;
+and some updates will be needed there.
+
+Each project should include:
+
+- Files that can be downloaded from github to provide some basic configuration with an explanation of what is provided in these files (that they would have to do themselves if they wanted to implement this functionality.
+- Specification in GitHub for what needs to be installed to provide a viable environment for the project, including integrations that must be included, configurations that must be set, and so forth.
+- Excellent information about how to run the sample project and do the exercises, including context for each step, what to look for to determine success and understand the output.  The goal is to begin to educate the user about how to get the most out of Keptn
+
+#### Separate guides for roles
+
+We want to introduce separate documentation for different Keptn user roles: 
+administrators, end users and developers.
+We suggest to work together to regroup the content so that the new structure is introduced.
+
+#### New reference pages
+
+Rewrite reference pages for files to conform to the structure proposed in the [Structure of reference pages](https://docs.google.com/document/d/1rmSGUFsnYuNc_BGHGmkW-9iB3hrOYbMeA-mJmMOjGRk/edit?usp=sharing) proposal.
+We are currently creating reference pages for critical files by capturing information in existing doc files, source code README files, etc.
+This project would be to restructure and expand the information in these files to create proper reference pages.
+It might also include relocating some information in existing files to a more appropriate location.
+
+Same for other standard documentation independent from version.
```

Comment:
```suggestion
Same for other standard documentation independent from the version.
```

Reply:
Is it "independent from the version" or is it "written to be relevant for all versions"?  The point is to document differences between versions rather than expect the reader to do textural analysis on the docs for different versions to determine when changes occur.